project-base/storefront/components/Pages/CategoryDetail/AdvancedSeoCategories.tsx

```diff
@@ -16,7 +16,7 @@ export const AdvancedSeoCategories: FC<AdvancedSeoCategoriesProps> = ({ readyCat
             <div className="mb-3 mt-6 break-words font-bold text-dark lg:text-lg">{t('Favorite categories')}</div>
             <SimpleNavigation
                 itemClassName={simpleNavigationItemTwClass}
-                linkType="category"
+                linkType="seo_category"
```

Comment:
Not sure why we need a separate type for SEO categories? What benefit does it bring us?